### PR TITLE
feat: support default exports for controllers

### LINKED ISSUE #372: [SUGGESTION] Problem with default exports "Controller X does not have method Y"
**Is your feature request related to a problem? Please describe.**
When using default export in a controller the library is not able to retrieve the requested method, and instead throws an error of Problem when exporting from default module "Controller X does not have method Y"

Currently it only supports named exports

**Describe the solution you'd like**
Support both named exports and a default export that contains the method requested

**Describe alternatives you've considered**
I can of course do named exports for each method in my module, but that would be a nightmare when adding new methods

**Additional context**
I should mention here that I'm using awilix library for dependency injection, which means that I don't have a "direct" module definition. So my controller file looks something like this

```
import container from 'container.js';

const myController = container.resolve('myController');

export default myController;
```
So, I could do something like
```
export const method1 = myController.method1;
export const method2 = myController.method2;
export const method3 = myController.method3;
...
```
But you see how it'd become hell to maintain really quick

### PR DESCRIPTION
**Pull request by webnator**
### Initial checks

* [x] Have you followed the guidelines in our Contributing document?
* [x] Have you checked to ensure there aren't other open [Pull Requests](../../../pulls) for the same update/change?
* [x] Have you linked an issue to this pull request? (Create one if it does not exist)
* [x] Have you used [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/) format?

#### Description
Support default exports in controllers [#372](https://github.com/oas-tools/oas-tools/issues/372)

#### Implementation details
Implemented in `src/middleware/native/oas-router.js`
Basically if method is not found, try to look for it in `default` property
Added associated test

### CODE REVIEW

**Comment by JDGuillaume** [comment]
Would love to see this pull request merged. Currently running into the issue solved by these changes.
